cli/src/test/java/com/devonfw/tools/ide/context/AbstractIdeTestContext.java

```diff
@@ -88,8 +88,8 @@ public IdeProgressBar prepareProgressBar(String taskName, long size) {
 
     IdeProgressBarTestImpl progressBar = new IdeProgressBarTestImpl(taskName, size);
     IdeProgressBarTestImpl duplicate = this.progressBarMap.put(taskName, progressBar);
-    // If we have multiple downloads, we may have an existing "Downloading" key
-    if (!taskName.equals("Downloading")) {
+    // If we have multiple downloads or unpacking, we may have an existing "Downloading" or "Unpacking" key
```

Comment:
Not sure if that's wrong behavior, but if not put this condition then the tests for extracting tar files don't work. Probably because the tests run simultaniously? Could be `@BeforeEach` useful in this case?

Reply:
Was added long time ago by @MustaphaOuchen 
I also would need to debug into this. Your change makes sense according to the existing workaround/hack.
However, IMHO the test progress bar should be improved but this can be done isolated from this PR so IMHO this change is fine and we can resolve this.